test_scripts/Polices/Policy_Table_Update/023_ATF_P_TC_PoliciesManager_changes_status_to_UP_TO_DATE.lua

```diff
@@ -0,0 +1,187 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] PoliciesManager changes status to “UP_TO_DATE”
+-- [HMI API] OnStatusUpdate
+-- [HMI API] OnReceivedPolicyUpdate notification
+--
+-- Description:
```

Comment:
Please check description, maybe it should be more like:
`PoliciesManager must change the status to “UP_TO_DATE” and notify HMI with OnStatusUpdate("UP_TO_DATE") right after successful validation of received PTU .`